tools/lkl/include/lkl_host.h

```diff
@@ -47,12 +47,21 @@ struct lkl_dev_blk_ops {
 	int (*request)(union lkl_disk disk, struct lkl_blk_req *req);
 };
 
+struct lkl_netdev {
```

Comment:
@liuyuan10 Because `lkl_netdev` needs to have an `lkl_thread_t` member, but `host_ops.h` isn't available in `lkl.h`. I guess it could be solved by adding an `include` to `lkl.h`, but I feel like there is a lot going on in `lkl.h` as it is, anyway. I think originally I needed to move it to `virtio.h` because I put a `virtio_net_dev` inside of an `lkl_netdev` and there was no other way to get at that type from there. As it is now I don't think there's a hugely compelling reason to do it one way or the other. Let me know if you think it should move back.